src/redux/actions/concepts/addExistingConcepts/index.js

```diff
@@ -0,0 +1,20 @@
+import { notify } from 'react-notify-toast';
+import instance from '../../../../config/axiosConfig';
+import { ADD_EXISTING_CONCEPTS } from '../../types';
+import { isSuccess } from '../../globalActionCreators';
+
+const addExistingConceptsAction = data => async (dispatch) => {
+  const typeName = localStorage.getItem('typeName');
+  const dictionaryId = localStorage.getItem('dictionaryId');
+  const userType = localStorage.getItem('type');
+  const url = `${userType}/${typeName}/collections/${dictionaryId}/references/`;
+  const payload = await instance.put(url, data);
+  dispatch(isSuccess(payload.data, ADD_EXISTING_CONCEPTS));
+  const conceptName = localStorage.getItem('conceptName');
+  if (payload.data[0].added === true) {
+    notify.show(`Concept ${conceptName} successfully added to your dictionary`, 'success', 3000);
```

Comment:
Do you have a test for this?

Reply:
yes, I have that's where I check for the key added === true, however, I customized the message to what the context a user can understand because the message from the API couldn't be easily understood by the user and I received feedback that It's better I customize it.